esbuild

```diff
@@ -9,9 +9,7 @@ node_modules/.bin/esbuild "$@"        \
     --bundle                          \
     --entry-names=[dir]/[name]-[hash] \
     --format=esm                      \
-    --inject:js/_inject.js            \
```

Comment:
Can we avoid dropping this? Really not a fan of having to manually import the JSX builder into every file. Happy to have the filenames be `.tsx` so as to not confuse tooling.

Reply:
Fixed; turned out to be easier than I expected